prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,61 @@
+function isAdult(age) {
+  return age < 18 ? false : true;
+}
+
+const isAdultResult = isAdult(17);
+console.log(isAdultResult);
+
+function didStudentPass(score) {
+  return score < 70 ? false : true;
+}
+
+const didStudentPassResult = didStudentPass(70);
+console.log(didStudentPassResult);
+
+function gradeCalculator(score) {
+  return score < 0
+    ? "How'd you even do that? F-"
+    : score < 60
+    ? 'F'
+    : score < 70
+    ? 'D'
+    : score < 80
+    ? 'C'
+    : score < 90
+    ? 'B'
+    : score <= 100
+    ? 'A'
+    : 'A++';
+}
```

Comment:
Totally valid, however if you have so many conditionals to check for, most people will choose to use `else..if`'s. Ternary's really shine when you only have one condition, and you want to handle both cases. As soon as you start needing to check for multiple conditions, people find `else..if` statements easier to read.

Reply:
Sounds good!